### ISSUE TITLE
It seems that it isn't compatible with a specific game app.

### ISSUE THREAD

**Issue by i1itione** [Magisk-Modules-Repo/movecert#3]
I use Charles for https proxying, I noticed if I use this module to move Charles' cert into /system, there is a game app will not trust the cert, but if I move the cert into /system directly (I disabled dm-verity so that's fine), the game app will trust the cert normally.

Is there any difference between using this module for moving cert and moving cert directly? 

Thanks, and sorry for my English.

**Comment by yochananmarqos** [comment]
No idea about Charles. Sorry, somehow I never noticed you created this issue.

**Comment by techIndia-hacker** [comment]
Yes there is a difference in moving the certificate directly and by this module. Google system apps don't trust certificates moved by this module but trusted them when moved directly in system cacerts in Android 10. But in most devices u can't put directly. So this module was created to do it systemlessly. But it does not work as Google system apps detect it somehow
